code/cmdline/cmdline.cpp

```diff
@@ -28,42 +29,36 @@
 #include "starfield/starfield.h"
 
 #ifdef _WIN32
-#include <io.h>
 #include <direct.h>
+#include <io.h>
```

Comment:
As much as I appreciate proper formatting, changing more than 100 lines for a (I guess) 2 line change is a bit excessive.

Reply:
It's your clang-format settings!

Reply:
I know but please only format the lines that you changed or otherwise tracking down what actually changed is a big hassle.